### PR TITLE
TELCODOCS-1862 - remove EOL AMQP docs for 4.16+

### PR DESCRIPTION
**Pull request by aireilly**
AMQP is EOL and no longer supported for PTP events transport.

Reviewers: This PR removes content only. No new content is added in this PR.

Version(s):
4.17

Issue:
https://issues.redhat.com/browse/TELCODOCS-1862

Link to docs preview:
https://80692--ocpdocs-pr.netlify.app/openshift-enterprise/latest/edge_computing/policygenerator_for_ztp/ztp-advanced-policygenerator-config.html
https://80692--ocpdocs-pr.netlify.app/openshift-enterprise/latest/edge_computing/policygentemplate_for_ztp/ztp-advanced-policy-config.html
https://80692--ocpdocs-pr.netlify.app/openshift-enterprise/latest/networking/ptp/ptp-cloud-events-consumer-dev-reference.html
https://80692--ocpdocs-pr.netlify.app/openshift-enterprise/latest/networking/ptp/using-ptp-events.html

QE review:
- [ ] QE has approved this change.

### CODE REVIEW

File: modules/cnf-about-ptp-and-clock-synchronization.adoc [hunk 1]
```diff
@@ -10,11 +10,9 @@ Loss of PTP synchronization is a critical error for a RAN network. If synchroniz
 
 Event notifications are available to vRAN applications running on the same DU node. A publish/subscribe REST API passes events notifications to the messaging bus. Publish/subscribe messaging, or pub-sub messaging, is an asynchronous service-to-service communication architecture where any message published to a topic is immediately received by all of the subscribers to the topic.
 
-The PTP Operator generates fast event notifications for every PTP-capable network interface. You can access the events by using a `cloud-event-proxy` sidecar container over an HTTP or Advanced Message Queuing Protocol (AMQP) message bus.
+The PTP Operator generates fast event notifications for every PTP-capable network interface. You can access the events by using a `cloud-event-proxy` sidecar container over an HTTP message bus.
```
Comment (jzding) [inline modules/cnf-about-ptp-and-clock-synchronization.adoc:13]: `over an HTTP message bus` -> `over HTTP transport`

**Review by jzding — APPROVED** [review]
/lgtm

**Comment by michaelryanpeter** [comment]
lgtm

/label peer-review-done
/remove-label peer-review-in-progress
/remove-label peer-review-needed

**Pull request merged**